pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -223,7 +223,53 @@ func (m *kubeGenericRuntimeManager) Status() error {
 // specifies whether the runtime returns all containers including those already
 // exited and dead containers (used for garbage collection).
 func (m *kubeGenericRuntimeManager) GetPods(all bool) ([]*kubecontainer.Pod, error) {
-	return nil, fmt.Errorf("not implemented")
+	pods := make(map[kubetypes.UID]*kubecontainer.Pod)
```

Comment:
Then it's just the same as getpods today :stuck_out_tongue: 

I am not super concerned about the performance as long as it's not too back, since for the long term we will use event system.